doc/user_manual/OutStreamSystem.tex

```diff
@@ -880,6 +879,11 @@ \subsubsection{Plot input structure \label{sec:PlotInputStructure}}
 \end{itemize}
 %
 
+One may also specify a \xmlNode{legend} tag that will place a legend on the plot.
+The legend accepts the same arguments as the matplotlib interface for creating a
+legend (refer to the \href{http://matplotlib.org/api/pyplot_api.html#matplotlib.pyplot.legend}{matplotlib API} for details.
+Note, that only primitive datatypes are accepted in the raven input file so values like bbox\_transform, prop, and handler\_map are not available through the raven interface).
+
```

Comment:
I attempted to do this in the latest commit. The result looks like this:

![legendtext](https://cloud.githubusercontent.com/assets/8671026/25291180/3a3c853e-268e-11e7-8cc2-992858ce3315.png)